src/applications/appeals/996/tests/hlr.cypress.helpers.js

```diff
@@ -26,68 +26,20 @@ export const fixDecisionDates = (data = []) => {
         ...issue.attributes,
         approxDecisionDate: newDate,
       },
-      [SELECTED]: false,
+      [SELECTED]: unselected ? false : issue[SELECTED],
     };
   });
 };
 
 const date = getDate({ offset: { months: -2 } });
 
-export const mockContestableIssues = {
-  data: [
-    {
-      id: null,
-      type: 'contestableIssue',
-      attributes: {
-        ratingIssueReferenceId: '142926',
-        ratingIssueProfileDate: date,
-        ratingIssueDiagnosticCode: '6260',
-        ratingIssueSubjectText: 'Tinnitus',
-        ratingIssuePercentNumber: '0',
-        description:
-          'Service connection for Tinnitus is granted with an evaluation of 0 percent effective September 25, 2019.',
-        isRating: true,
-        latestIssuesInChain: [
-          {
-            id: null,
-            approxDecisionDate: date,
-          },
-        ],
-        decisionIssueId: null,
-        ratingDecisionReferenceId: null,
-        approxDecisionDate: date,
-        rampClaimId: null,
-        titleOfActiveReview: null,
-        sourceReviewType: null,
-        timely: true,
-      },
-    },
-    {
-      id: null,
-      type: 'contestableIssue',
-      attributes: {
-        ratingIssueReferenceId: '142927',
-        ratingIssueProfileDate: date,
-        ratingIssueDiagnosticCode: '9411',
-        ratingIssueSubjectText: 'PTSD',
-        ratingIssuePercentNumber: '30',
-        description:
-          'Service connection for PTSD is granted with an evaluation of 30 percent effective March 5, 2019.',
-        isRating: true,
-        latestIssuesInChain: [
-          {
-            id: null,
-            approxDecisionDate: date,
-          },
-        ],
-        decisionIssueId: null,
-        ratingDecisionReferenceId: null,
-        approxDecisionDate: date,
-        rampClaimId: null,
-        titleOfActiveReview: null,
-        sourceReviewType: null,
-        timely: true,
-      },
+export const mockContestableIssues = ({ contestedIssues }) =>
+  contestedIssues.map(issue => ({
+    id: null,
+    type: 'contestableIssue',
+    attributes: {
+      ...issue.attributes,
+      approxDecisionDate: date,
     },
-  ],
-};
+    [SELECTED]: false,
```

Comment:
Does this one not also get the same pattern: `[SELECTED]: unselected ? false : issue[SELECTED],`?

Reply:
We're using this in the `cy.intercept` as a response and we don't want to pre-select the options because we want to test the checkboxes. In the contestable issue page hook, we use the data from `@testData` (which is directly from the mock data, maximal or minimal) we use that selected state to check the checkboxes. This function can really be removed and set the intercept with an empty object.

Reply:
Well, since I noticed that the stress test was failing - I found out that the keyboard e2e test was the culprit. I'm not sure how all the e2e tests passed in the first place. Anyway, I made some updates to the e2e tests.